tools/api-markdown-documenter/src/utilities/index.ts

```diff
@@ -3,4 +3,7 @@
  * Licensed under the MIT License.
  */
 
+// All of the utilities here are meant to be used outside of this directory.
+// eslint-disable-next-line no-restricted-syntax
```

Comment:
why do we export it like this?

Reply:
It's just so we don't have to list everything here explicitly. The `ApiItemUtilities` module is exporting general purpose utilities that can be used anywhere else in the pacakge. We scope down which of these members gets exported by the package elsewhere, but internally we can export everything here.